apps/web/src/components/Home/Timeline/EventType/index.tsx

```diff
@@ -20,7 +20,7 @@ interface ActionTypeProps {
 const ActionType: FC<ActionTypeProps> = ({ feedItem }) => {
   const publication = feedItem.root;
   const isComment = publication.__typename === 'Comment';
-  const showThread = isComment || (feedItem.comments?.length ?? 0 > 0);
+  const showThread = isComment || (feedItem.comments?.length ?? 0) > 0;
```

Comment:
We dont have any ops to do first, so we don't need to follow BODMAS rule here 🙇🏼

Reply:
According to [JavaScript operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence), `(feedItem.comments?.length ?? 0 > 0)` is equivalent to `(feedItem.comments?.length ?? (0 > 0))`, which does not return a boolean if `length` exists. For example,

```
> 1 ?? 0 > 0
1
> (1 ?? 0) > 0
true
```

Reply:
If returning a number is expected, the expression could also be simplified into `isComment || (feedItem.comments?.length ?? false)`.

@bigint What do you think?

Reply:
Ahh this makes sense 🙏🏼 

Reopened the PR 💟 

Thanks for explaining